website/src/pages/portal.mdx

```diff
@@ -83,3 +85,9 @@ Any content you want to render inside of the portal.
 _Type_: `string` default: `reach-portal`
 
 The DOM element type to render.
+
+##### Portal nodeContainer
+
+_nodeContainer_: `HTMLElement` default: `document.body`
+
+The DOM element container of the portal. If not set the portal will be appended to the body of the document.
```

Comment:
Update the prop name and type here as well, and tweak to read "The DOM **node** container..."